### ISSUE TITLE
mapping documentation missing

### ISSUE THREAD

**Issue by pylgrim73** [omoponfhir/omoponfhir-omopv5-r4-mapping#5]
Hi All,

I started using the project but I'm alittle bit lost about the mapping performed. I mean it is coded within java classes and I don't understand if such mappings refer, somehow, official mapping intiatives or, even if not official, at least supported by a huge community.
For example I see AlergyIntollerance is mapped to OMOP observation etc etc...is there a documentation about the choice?
Basically I would like to understand if the implemented mappings are related to a documented conceptual mappings (e.g. by pdf, excel or whatever) available somewhere.
Thx

**Comment by myungchoi** [comment]
It's on our roadmap. We are being short of staff to do this mapping to be up to dated in the document.
